csrc/executor.h

```diff
@@ -312,6 +318,20 @@ class TORCH_CUDA_CU_API FusionExecutor : public NonCopyable {
       const LaunchParams& new_launch_params,
       const CompileParams& new_compile_params);
 
+  flatbuffers::Offset<serde::ExecutorEntry> serialize(
```

Comment:
Could you add a comment on the purpose of `private` overloads of `serialize` and `deserialize` methods.

Reply:
Just FYI,
```cpp
// ExecutorEntry and GlobalBufferInfo are internal POD structs for the FusionExecutor class.
// We define private serialize and deserialize methods in FusionExecutor to keep this property.
```